app/server/openshift/src/main/java/io/syndesis/server/openshift/OpenShiftConfigurationProperties.java

```diff
@@ -57,6 +57,8 @@
 
     private int integrationLivenessProbeInitialDelaySeconds;
 
+    private String urlTo3scale;
```

Comment:
Perhaps

```suggestion
    private String _3scaleManagementUrl;
```

Reply:
I renamed it to managementUrlFor3scale to avoid the use of '_', which is not liked by go (can only export variables starting with [A-Z]), and keep same names for go and java code.